<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaleWidget</class>
 <widget class="QWidget" name="SaleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>937</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sale</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/resources.qrc">
    <normaloff>:/icons/sale.svg</normaloff>:/icons/sale.svg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="customerLabel">
       <property name="text">
        <string>Customer:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="searchCustomersLineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="searchCustomerBtn">
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="searchCustomersTableView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="productLabel">
       <property name="text">
        <string>Product:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="searchProductsLineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="addProductBtn">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="searchProductsTableView">
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="totalLabel">
       <property name="text">
        <string>Total:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="discountLabel">
       <property name="text">
        <string>Discount:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="discountSpinBox">
       <property name="maximum">
        <double>99999.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="saleProductsTableView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelBtn">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveBtn">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>searchCustomersLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>searchCustomerBtn</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>812</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>876</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>searchCustomerBtn</sender>
   <signal>clicked()</signal>
   <receiver>SaleWidget</receiver>
   <slot>searchCustomers()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>897</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>930</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>searchProductsLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>SaleWidget</receiver>
   <slot>searchProducts()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>791</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>933</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>searchProducts()</slot>
  <slot>searchCustomers()</slot>
 </slots>
</ui>
